src/meshapi/urls.py

```diff
@@ -28,6 +28,7 @@
     path("mapdata/links/", views.MapDataLinkList.as_view(), name="meshapi-v1-map-data-links"),
     path("mapdata/sectors/", views.MapDataSectorList.as_view(), name="meshapi-v1-map-data-sectors"),
     path("geography/whole-mesh.kml", views.map_kml, name="meshapi-v1-geography-whole-mesh-kml"),
+    path("update-panoramas/", views.update_panoramas_from_github, name="meshapi-v1-update-panoramas"),
```

Comment:
Hm maybe we should do this as an async background task instead of exposing it as an HTTP endpoint?

Reply:
Is there any reason we'd need to do this on-demand?

Reply:
We could also just schedule it to run hourly with celery